acceptance/acceptance_test.go

```diff
@@ -450,11 +449,12 @@ func CopyDir(src, dst string, inputs, outputs map[string]bool) error {
 	})
 }
 
-func ListDir(t *testing.T, src string) ([]string, error) {
+func ListDir(t *testing.T, src string) []string {
 	var files []string
 	err := filepath.Walk(src, func(path string, info os.FileInfo, err error) error {
 		if err != nil {
-			return err
+			t.Errorf("Error when listing %s: path=%s: %s", src, path, err)
```

Comment:
Could you include a comment on why neither needs to `t.FailNow()`?

The returned value will be incomplete and additional asserts will run, potentially masking this underlying issue.

Reply:
I tried :)

> If there are unreadable files in a directory, raise an error but continue with further diagnostics, because the answer is in the script output.

The output comparison is happening after this call which includes output.txt which includes errors printed by commands which include explanation why a given file cannot be read.

> The returned value will be incomplete and additional asserts will run, potentially masking this underlying issue.
It is not masked, everything will be printed. We do not know which one is more relevant in advance. In my case latter was more relevant.

Reply:
https://github.com/databricks/cli/pull/2216/commits/7d15082a399e6003e9425a4afaefe9edbc5712f3